.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
 .idea
+
+__amy_scratch
```

Comment:
You probably didn't mean to commit this…

There's a local ignore file in _.git/info/exclude_ in your checked-out repo directory — if you add `__amy_scratch` to that, it'll be ignored just for you and there's no risk of checking it in.

(Also use _~/.config/git/ignore_ for locally ignoring things in all the repos on your machine.)

Reply:
Thanks John. Good tip